tests/pass/mpsc.rs

```diff
@@ -1,3 +1,4 @@
+//@only-target-linux: support for tokio only on linux
```

Comment:
This isn't a tokio test, please don't change it. :)

Reply:
Yea I just realised I edited the wrong mpsc.rs, I renamed the original one so it is less confusing :')